src/lib.rs

```diff
@@ -80,7 +80,7 @@ pub fn process_file(
     );
     let parser = Parser::new_ext(
         &input,
-        Options::ENABLE_TASKLISTS | Options::ENABLE_STRIKETHROUGH,
+        Options::ENABLE_TASKLISTS | Options::ENABLE_STRIKETHROUGH | Options::ENABLE_TABLES,
```

Comment:
Don't we also need to update all tests which parse markdown to enable tables?  So as to check that we don't break anything?